packages/select/package.json

```diff
@@ -1,40 +1,39 @@
 {
-  "name": "@xcritical/select",
-  "version": "0.0.0",
-  "description": "XCritical Select Component",
-  "main": "src/index.js",
-  "scripts": {
-    "test": "exit 0",
-    "compile": "gulp --cwd . compile",
-    "build": "yarn clean && yarn compile",
-    "clean": "rimraf dist lib .publish"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com:xcritical-software/xc-front-kit.git"
-  },
-  "keywords": [
-  ],
-  "author": "XCritical Software",
-  "license": "MIT",
-  "homepage": "https://github.com/xcritical-software/xc-front-kit/tree/develop/packages/select#readme",
-  "publishConfig": {
+	"name": "@xcritical/select",
```

Comment:
Problem with indentation, 4 instead of 2.

Reply:
![image](https://user-images.githubusercontent.com/1057075/68398010-20c0b600-0185-11ea-9238-016de11a7253.png)
Nope. it's all right